--- ehealth/openncp-docker/openncp-openatna/openatna.env	2026-02-17 15:46:44
+++ ehdsi/NCP/openncp-openatna/openatna.env	2025-02-21 09:09:49
@@ -3,6 +3,6 @@
 ##############################################################################
 CONNECTOR_PORT=${NCP_OPENATNA_CONNECTOR_PORT}
 CONNECTOR_SECURE_PORT=${NCP_OPENATNA_CONNECTOR_SECURE_PORT}
-KEY_ALIAS=gazelle.ncp-sc.openncp.dg-sante.eu
-KEYSTORE_PASSWORD=gazelle
-TRUSTSTORE_PASSWORD=changeit
+KEY_ALIAS=${TLS_KEY_ALIAS}
+KEYSTORE_PATH=${TLS_KEYSTORE_PATH}
+TRUSTSTORE_PATH=${TLS_TRUSTSTORE_PATH}
