--- ehealth/openncp-docker/openncp-trc-sts/.database.env	2024-10-08 16:04:22.819373478 +0200
+++ ehdsi/NCP/openncp-trc-sts/.database.env	2024-10-30 09:26:17.375979817 +0100
@@ -3,8 +3,6 @@
 # !!! DO NOT CHANGE THIS !!!                                                            #
 #########################################################################################
 DB_PORT=${DB_INTERNAL_PORT}
-DB_USER=${DB_ROOT_USER}
-DB_PWD=${DB_ROOT_PASSWORD}
 DB_TYPE=${DB_TYPE}
 DB_HOST=${DB_HOST}
 DATASOURCE_CLASS_NAME=${DATASOURCE_CLASS_NAME}
