--- ehealth/openncp-docker/openncp-translations-and-mappings/translations-and-mappings.database.env
+++ ehdsi/NCP/openncp-translations-and-mappings/translations-and-mappings.database.env
@@ -3,8 +3,6 @@
 # !!! DO NOT CHANGE THIS !!!                                                            #
 #########################################################################################
 DB_PORT=${DB_INTERNAL_PORT}
-DB_USER=${DB_ROOT_USER}
-DB_PWD=${DB_ROOT_PASSWORD}
 DB_TYPE=${DB_TYPE}
 DB_HOST=${DB_HOST}
 DATASOURCE_CLASS_NAME=${DATASOURCE_CLASS_NAME}
