--- ehealth/openncp-docker/openncp-translations-and-mappings/translations-and-mappings.env	2026-02-17 15:46:44
+++ ehdsi/NCP/openncp-translations-and-mappings/translations-and-mappings.env	2025-02-21 09:09:49
@@ -3,6 +3,6 @@
 ##############################################################################
 CONNECTOR_PORT=${NCP_TRANSLATIONS_MAPPINGS_CONNECTOR_PORT}
 CONNECTOR_SECURE_PORT=${NCP_TRANSLATIONS_MAPPINGS_CONNECTOR_SECURE_PORT}
-KEY_ALIAS=gazelle.ncp-sp.openncp.dg-sante.eu
-KEYSTORE_PASSWORD=gazelle
-TRUSTSTORE_PASSWORD=changeit
+KEY_ALIAS=${TLS_PROVIDER_KEY_ALIAS}
+KEYSTORE_PATH=${TLS_PROVIDER_KEYSTORE_PATH}
+TRUSTSTORE_PATH=${TLS_TRUSTSTORE_PATH}
