--- ehealth/openncp-docker/ncp_a/ncpa.database.env	2026-02-17 15:46:44
+++ ehdsi/NCP/ncp_a/ncpa.database.env	2025-02-21 09:09:49
@@ -3,8 +3,6 @@
 # !!! DO NOT CHANGE THIS !!!                                                            #
 #########################################################################################
 DB_PORT=${DB_INTERNAL_PORT}
-DB_USER=${DB_ROOT_USER}
-DB_PWD=${DB_ROOT_PASSWORD}
 DB_TYPE=${DB_TYPE}
 DB_HOST=${DB_HOST}
 DATASOURCE_CLASS_NAME=${DATASOURCE_CLASS_NAME}
