--- ehealth/openncp-docker/ncp_b/ncpb.env	2026-02-17 15:46:44
+++ ehdsi/NCP/ncp_b/ncpb.env	2025-02-21 09:09:49
@@ -3,7 +3,7 @@
 ##############################################################################
 CONNECTOR_PORT=${NCP_B_SERVER_CONNECTOR_PORT}
 CONNECTOR_SECURE_PORT=${NCP_B_SERVER_CONNECTOR_SECURE_PORT}
-KEY_ALIAS=gazelle.ncp-sc.openncp.dg-sante.eu
-KEYSTORE_PASSWORD=gazelle
-TRUSTSTORE_PASSWORD=changeit
+KEY_ALIAS=${TLS_KEY_ALIAS}
+KEYSTORE_PATH=${TLS_KEYSTORE_PATH}
+TRUSTSTORE_PATH=${TLS_TRUSTSTORE_PATH}
 CLIENT_AUTH=true
