--- a/country-a-service/cda-generator/src/main/java/dk/sundhedsdatastyrelsen/ncpeh/cda/EPrescriptionPdfMapper.java
+++ b/country-a-service/cda-generator/src/main/java/dk/sundhedsdatastyrelsen/ncpeh/cda/EPrescriptionPdfMapper.java
@@ -3,6 +3,7 @@ package dk.sundhedsdatastyrelsen.ncpeh.cda;
 import dk.sundhedsdatastyrelsen.ncpeh.cda.model.Author;
 import dk.sundhedsdatastyrelsen.ncpeh.cda.model.EPrescriptionL3;
 import dk.sundhedsdatastyrelsen.ncpeh.cda.model.EPrescriptionPdf;
+import dk.sundhedsdatastyrelsen.ncpeh.cda.model.EhdsiUnit;
 import dk.sundhedsdatastyrelsen.ncpeh.cda.model.Patient;
 import dk.sundhedsdatastyrelsen.ncpeh.cda.model.PdfField;
 import org.apache.commons.lang3.StringUtils;
@@ -35,11 +36,10 @@ public class EPrescriptionPdfMapper {
     private static String packageLine(EPrescriptionL3 model) {
         var quantity = model.getPackageQuantityLong();
         var packagePlural = quantity > 1 ? "pakker" : "pakke";
-        var unit = """
-            switch (model.getProduct().getSize().getUnit()) {
+        var unit = switch (model.getProduct().getSize().getUnit()) {
             case EhdsiUnit.WithCode u -> u.getCode();
             case EhdsiUnit.WithTranslation u -> u.getTranslation();
-        }""";
+        };
         return String.format(
             "%s %s à %s %s %s",
             quantity,
